js/script.js

```diff
@@ -1,65 +1,94 @@
-const form = document.querySelector('#form');
-const data = document.querySelector('#data');
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 
-function addNewBookData() {
-  const bookTitle = form.title.value;
-  const bookAuthor = form.author.value;
+  static removeBook(author) {
+    const books = Store.getBooks();
 
-  const book = {
-    title: bookTitle,
-    author: bookAuthor,
-  };
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
+    });
 
```

Comment:
- [ ] Great job so far 👏🏻 However, I noticed that if two or three books have the same title name and author name, and when we delete one of them, all other books are also deleted and it is because you delete a book by checking the book which has the same author with the selected one. Please correct this. 🤔  I will suggest you to use a unique book id instead of author's name to avoid this 👍🏻 

eg: 
```javascript
  removeBook(bookID) {
    // bookID can be any other property, e.g. title or author, in as much is it unique
    this.books = this.books.filter(({ id }) => bookID !== id); 
    return this.books; 
  }
```

Reply:
@BenMukebo Thanks for your kind review, please I really don't know how to go about the changing of the author to books